share/lib/python/neuron/__init__.py

```diff
@@ -1051,8 +1054,16 @@ def mark(self, segment, marker="or", **kwargs):
                     return self
 
                 def _do_plot(
-                    self, val_min, val_max, sections, variable, cmap=cm.cool, **kwargs
-                ):
+                    self,
+                    val_min,
+                    val_max,
+                    sections,
+                    variable,
+                    mode,
+                    line_width=2,
+                    cmap=cm.cool,
+                    **kwargs,
+                ):  ## added show_color and line_width here
```

Comment:
I hate to delay this any further, but can you remove these `## added...` comments? (Here and elsewhere). Git will keep track of what has been changed.